freqtrade/optimize/backtesting.py

```diff
@@ -476,6 +539,21 @@ def _enter_trade(self, pair: str, row: List) -> Optional[LocalTrade]:
                 buy_tag=row[BUY_TAG_IDX] if has_buy_tag else None,
                 exchange='backtesting',
             )
+            order = Order(
```

Comment:
i do have "a few" performance concerns about the usage of the `Order` object directly in backtesting as it's an sqlalchemy object (using an attribute there has a callstack as deep as from here to the Moon and back - which has quite severe performance implications).

For the moment, keep it as is - but we will need to consider changing this to a "LocalOrder" object eventually (so this is also a reminder for myself).

Reply:
I did refactor a LocalOrder and measured the performance difference and there was exactly less than a second difference for backtesting 120 pairs with 2 months of data. I'm sure there might be 'some' benefit to it but I reverted it as that would've caused too many changes for unnoticeable benefit.